### PR TITLE
linux/[l-m]: add more information link 

### PR DESCRIPTION
**Pull request by schneiderl**
Follow up to #6227

I felt stupid when I realized I was doing all that manually. I'll write a script to automatically add those links to pages tomorrow. 

see you there.

### CODE REVIEW

**Comment by marchersimon** [comment]
@schneiderl which script are you thinking about? We already have a [script](https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py) for adding a link to all translated pages and there's currently a PR (#6084) to automatically use the `manned.org` link

File: pages/linux/mac2unix.md
Comment (marchersimon) [inline pages/linux/mac2unix.md:5]:
```suggestion
> More information: <https://waterlan.home.xs4all.nl/dos2unix.html>.
```

**Review by marchersimon — COMMENTED** [review]
Just one small suggestion

**Comment by schneiderl** [comment]
@marchersimon I see. I was a bit away over the past couple of months so I'm not fully up-to-date on what happened to tldr in the meantime. Thank you for the information.

**Review by marchersimon — APPROVED** [review]
Thanks @schneiderl

**Comment by schneiderl** [comment]
Ping! 

Shall we merge this?

**Comment by marchersimon** [comment]
Sure, just waiting for a last approval.

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**
